### ISSUE TITLE
LookupError: unknown encoding: string_escape

### ISSUE THREAD

**Issue by nbecker** [erikrose/parsimonious#92]
run on py3.5:

```
# -*- coding: utf-8 -*-
from parsimonious.grammar import Grammar
grammar = Grammar(
    """
    bold_text  = bold_open text bold_close
    text       = ~"[A-Z 0-9]*"i
    bold_open  = "(("
    bold_close = "))"
    """)

gen = Grammar(
    """
    point = "."
    e = ~"[e]"i
    """)
```

```
In [3]: gen
Out[3]: 
---------------------------------------------------------------------------
LookupError                               Traceback (most recent call last)
/home/nbecker/.local/lib/python3.5/site-packages/IPython/core/formatters.py in __call__(self, obj)
    668                 type_pprinters=self.type_printers,
    669                 deferred_pprinters=self.deferred_printers)
--> 670             printer.pretty(obj)
    671             printer.flush()
    672             return stream.getvalue()

/home/nbecker/.local/lib/python3.5/site-packages/IPython/lib/pretty.py in pretty(self, obj)
    366                 if cls in self.type_pprinters:
    367                     # printer registered in self.type_pprinters
--> 368                     return self.type_pprinters[cls](obj, self, cycle)
    369                 else:
    370                     # deferred printer

/home/nbecker/.local/lib/python3.5/site-packages/IPython/lib/pretty.py in inner(obj, p, cycle)
    602         if basetype is not None and typ is not basetype and typ.__repr__ != basetype.__repr__:
    603             # If the subclass provides its own repr, use it instead.
--> 604             return p.text(typ.__repr__(obj))
    605 
    606         if cycle:

/home/nbecker/.local/lib/python3.5/site-packages/parsimonious/grammar.py in __repr__(self)
    139     def __repr__(self):
    140         """Return an expression that will reconstitute the grammar."""
--> 141         return "Grammar('%s')" % str(self).encode('string_escape')
    142 
    143 

LookupError: unknown encoding: string_escape
```

**Comment by bradst** [comment]
I can confirm this using 0.6.2 from PyPI and Python 3.5.2. The `string_escape` encoding is called `unicode_escape` in Python 3.

**Comment by lucaswiman** [comment]
This is fixed in #89: https://github.com/erikrose/parsimonious/pull/89/files#diff-b91e4f8d41712632724651a93807d646R151

**Comment by lonnen** [comment]
closing this as fixed. Thanks lucaswiman!
